common/vlog.h

```diff
@@ -25,12 +42,10 @@ namespace Carbon {
 // However, the streaming syntax has higher overhead and can inhibit inlining.
```

Comment:
Didn't you say you dropped support for the legacy syntax?

Reply:
Forgot to delete the comment, fixed!